prow/plugins.yaml

```diff
@@ -33,6 +33,7 @@ approve:
   - kubernetes/kubectl
   - kubernetes/kubernetes-template-project
   - kubernetes/minikube
+  - kubernetes/repo-infra
```

Comment:
This just specifies the configuration for the approve plugin. You'll need to enable it in the `plugins` section below as well by adding
```
  kubernetes/repo-infra:
  - approve
```

Reply:
done, thanks!